iothub/device/samples/convention-based-samples/TemperatureController/TemperatureControllerSample.cs

```diff
@@ -129,15 +129,28 @@ private async Task HandleTargetTemperatureUpdateRequestAsync(string componentNam
             _logger.LogDebug($"Property: Received - component=\"{componentName}\", [ \"{targetTemperatureProperty}\": {targetTemperature}°C ].");
 
             _temperature[componentName] = targetTemperature;
+            // Same feedback on serializing -- odd to do it here.
             IWritablePropertyResponse writableResponse = _deviceClient
                 .PayloadConvention
                 .PayloadSerializer
                 .CreateWritablePropertyResponse(_temperature[componentName], CommonClientResponseCodes.OK, version, "Successfully updated target temperature.");
 
-            var reportedProperty = new ClientPropertyCollection();
-            reportedProperty.AddComponentProperty(componentName, targetTemperatureProperty, writableResponse);
+            // My C# is rusty.  Is there a way to build these classes so you can use object initializers for this?
```

Comment:
Any time you implement Add() on a IDictionary<> or IList<> you can use the initializers